src/world/common/SetNpcColHeight.inc.c

```diff
@@ -0,0 +1,17 @@
+#include "common.h"
+
+static void SetNpcColHeight(ScriptInstance* script) {
+    Enemy* enemy = script->owner1.enemy;
+    Npc* npc = get_npc_unsafe(enemy->npcID);
+
+    if (npc->duration++ == 2) {
+        npc->collisionHeight = enemy->varTable[6];
+    }
+
+    if (npc->duration < enemy->varTable[5]) {
+        return;
```

Comment:
This too..if this is a script function, it should also take s32 isInitialCall, and it should be returning actual stuff. Maybe it's not a script function and I'm just confused.